core/views/browse.py

```diff
@@ -172,21 +174,18 @@ def issue_pages_rdf(request, lccn, date, edition):
 
 @cache_page(settings.DEFAULT_TTL_SECONDS)
 @vary_on_headers('Referer')
-def page(request, lccn, date, edition, sequence, words=None):
-    fragments = []
-    if words:
-        fragments.append("words=" + words)
-    qs = request.META.get('QUERY_STRING')
-    if qs:
-        fragments.append(qs)
-    if fragments:
-        path_parts = dict(lccn=lccn, date=date, edition=edition,
-                          sequence=sequence)
-        url = urlresolvers.reverse('chronam_page',
-                                   kwargs=path_parts)
-
-        return HttpResponseRedirect(url + "#" + "&".join(fragments))
+@deprecated
+def page_words(request, lccn, date, edition, sequence, words=None):
+    # for the case where we have ;words= in the url convert it to a fragment but
```

Comment:
I'd probably change this into a docstring and update the word a bit to include an example URL and a link to #126 for anyone wondering why this is necessary.